parser/prism/Translator.cc

```diff
@@ -1037,8 +1037,11 @@ unique_ptr<parser::Node> Translator::translate(pm_node_t *node) {
             translateMultiInto(params, requireds);
             translateMultiInto(params, optionals);
 
-            if (paramsNode->rest != nullptr)
-                params.emplace_back(translate(paramsNode->rest));
+            auto prismRestNode = paramsNode->rest;
+            // Implicit rest node in parameters don't need to be translated
```

Comment:
Could you please elaborate on the "why"? What was the behaviour before?

Reply:
Or do you think I should update it to be `can NOT be translated`?